internal/service/ipam/model_ipamsvc_fixed_address.go

```diff
@@ -0,0 +1,240 @@
+package ipam
+
+import (
+	"context"
+	"github.com/hashicorp/terraform-plugin-framework-timetypes/timetypes"
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	schema "github.com/hashicorp/terraform-plugin-framework/resource/schema"
```

Comment:
Redundant Alias. Probably should fix this in the generator
```suggestion
	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
```